tests/zquantum/core/circuits/conversions/pyquil_conversions_test.py

```diff
@@ -9,10 +9,33 @@
 )
 
 SYMPY_THETA = sympy.Symbol("theta")
+SYMPY_PHI = sympy.Symbol("phi")
+SYMPY_LAMBDA = sympy.Symbol("lambda")
 SYMPY_GAMMA = sympy.Symbol("gamma")
+
 QUIL_THETA = pyquil.quil.Parameter("theta")
 QUIL_GAMMA = pyquil.quil.Parameter("gamma")
 
+"""
+Note: Those differently named Symbols/Parameters are needed due
+to the manner the conversion logic abstracts the names of the Symbols
```

Comment:
I'm not sure I get it, could you elaborate? :)

Reply:
when exporting circuits with more than 1 parameter from z-quantum to pyquil, it changes the symbol names from (alpha, beta, gamma, etc...) to (theta_0, theta_1, theta_2, etc...), so to account for that, the name of PyQuil Parameters need to be adapted accordingly to pass the tests, since we are comparing code using assembly code.